packages/sdk/src/controllers/SubscriberController.ts

```diff
@@ -93,15 +93,29 @@ export class SubscriberController {
     };
 
     /**
-     * A listener on the general state of the document, gets triggered every time a change is done on the document.
+     * Listener on the general state of the document, gets triggered every time a change is done on the document.
      */
     onStateChanged = () => {
         const callBack = this.config.onStateChanged;
         callBack && callBack();
     };
 
     /**
-     * A listener on when the document is fully loaded.
+     * Listener on grafx token expiration.
+     * The callback should be the new string token. If the listener is not defined
+     * the http requests from the connectors will return 403 with no refetch
+     * of assets.
+     *
+     * Only grafx tokens are supported for now.
+     */
+    onTokenExpired = () => {
```

Comment:
If we expect this method to be used for more than just grafx tokens we should:

- Add connector id to body
- Rename this method to 'onAuthExpired' to be more in line with other auth types

Reply:
Not sure about the naming though :)